packages/auth/scripts/get-op-schema.sh

```diff
@@ -9,5 +9,6 @@ while [ -L "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 OUTDIR=$( cd -P "$( dirname "$SOURCE" )/../src/openapi" >/dev/null 2>&1 && pwd )
 
-curl -o "$OUTDIR/schemas.yaml" https://raw.githubusercontent.com/interledger/open-payments/fafd06ad0a3f205d25eec6ad5f388320e64abcd2/openapi/schemas.yaml
-curl -o "$OUTDIR/auth-server.yaml" https://raw.githubusercontent.com/interledger/open-payments/fafd06ad0a3f205d25eec6ad5f388320e64abcd2/openapi/auth-server.yaml
+# TODO: revert to using main once https://github.com/interledger/rafiki/issues/630 is resolved
+curl -o "$OUTDIR/schemas.yaml" https://raw.githubusercontent.com/wilsonianb/open-payments/b61bc0c84948386cac20d0e953a7e16de1e94471/openapi/schemas.yaml
+curl -o "$OUTDIR/auth-server.yaml" https://raw.githubusercontent.com/wilsonianb/open-payments/b61bc0c84948386cac20d0e953a7e16de1e94471/openapi/auth-server.yaml
```

Comment:
What changed in these schemas from `main` on https://github.com/interledger/open-payments?

Reply:
I think my comment is bad.
The versions from my fork include:
- https://github.com/interledger/open-payments/pull/211

Reply:
Actually, that's not my comment.
I just accidentally left in when resolving conflicts with `main`
https://github.com/interledger/rafiki/blame/7b911b17f5b567a47a2a5e671cfaa83e87e34d91/packages/auth/scripts/get-op-schema.sh

Reply:
I _thought_ it looked familiar...